### PR TITLE
(DOCSP-18526): September 8 2021 Release Notes

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
### Jira

- https://jira.mongodb.org/browse/DOCSP-18526

### Staged Changes (Requires MongoDB Corp SSO)

- [Release Notes](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-18526/release-notes/backend/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/release-notes/changelogs/backend/changelog-backend-2021.rst [hunk 1]
```diff
@@ -1,3 +1,16 @@
+.. _backend_20210908:
+
+02 September 2021 Release
```
Comment (nathan-contino-mongo) [inline source/release-notes/changelogs/backend/changelog-backend-2021.rst:3]:
```suggestion
08 September 2021 Release
```

File: source/release-notes/changelogs/backend/changelog-backend-2021.rst [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Increased request limit from 3000 requests per second to 5000 requests per second.
+- Allows users to store non-{+service-short+} files in the {+service-short+}
+  :ref:`app structure <app-configuration>`.
+- Updates to documents that do not match an application's :ref:`schema <realm-schemas>`
+  can now enable {+sync+} for those documents.
+
+- Github Code Deployment will now allow non-Realm files to be stored in the Realm App Structure
+Updates can now be used to correct documents that didn’t match Realm Sync Schema
```
Comment (nathan-contino-mongo) [inline source/release-notes/changelogs/backend/changelog-backend-2021.rst:13]:
```suggestion
```

File: source/release-notes/changelogs/backend/changelog-backend-2021.rst [hunk 3]
```diff
@@ -1,3 +1,14 @@
+.. _backend_20210908:
+
+08 September 2021 Release
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Increased request limit from 3000 requests per second to 5000 requests per second.
```
Comment (dacharyc) [inline source/release-notes/changelogs/backend/changelog-backend-2021.rst:6]: Just curious - is request limit a concept we have somewhere in the docs, and/or is there information about what counts as a request somewhere for the purpose of this limit increase? I found [Realm Requests in billing](https://docs.mongodb.com/realm/billing/#realm-requests) - do we know if this is what we're talking about? We might want to link somewhere relevant for users who want to know more about this.

**Review by dacharyc — APPROVED** [review]
LGTM, although I'm wondering whether we can link "request limit" to somewhere relevant for folks.

File: source/release-notes/changelogs/backend/changelog-backend-2021.rst (hunk 3 above)
Comment (nathan-contino-mongo) [reply source/release-notes/changelogs/backend/changelog-backend-2021.rst:6]: Good point, I'll link to the billing page for now. Drew mentioned that the realm requests change was marked docs-needed, so I think we'll add some additional information about realm request limits in the upcoming days. But for now, just the billing page link ought to do.
```suggestion
- Increased :ref:`request <billing-requests>` limit from 3000 requests per second to 5000 requests per second.
```

**Comment by drewdipalma** [comment]
LGTM

**Pull request merged**